### PR TITLE
render strikethrough

### PR DESCRIPTION
**Pull request by ellell**
Type: Minor
Review: @kesla @iefserge

### CODE REVIEW

File: lib/text.js [hunk 1]
```diff
@@ -9,13 +9,16 @@ const wrapBold = ({bold}, el) => bold ? <b>{el}</b> : el;
 
 const wrapItalic = ({italic}, el) => italic ? <i>{el}</i> : el;
 
+const wrapStrikethrough = ({strikethrough}, el) => strikethrough ? <s>{el}</s> : el;
```
Comment (danmakenoise) [inline lib/text.js]: I get that this is a clean one-liner, but I think as an outsider it makes this really confusing to have the arrow function AND the ternary in one line.

**Comment by danmakenoise** [comment]
1 Comment and then LGTM

File: lib/text.js (hunk 1 above)
Comment (ellell) [inline lib/text.js]: Discussed on slack and agreed this is ok in the whole context of the file.

File: lib/text.js [hunk 2]
```diff
@@ -9,13 +9,16 @@ const wrapBold = ({bold}, el) => bold ? <b>{el}</b> : el;
...
+
 const renderItem = (item) => {
   if (item.type === 'text') {
     let el = String(item.content || '');
     el = wrapItalic(item, el);
     el = wrapBold(item, el);
     el = wrapHref(item, el);
     el = wrapMark(item, el);
+    el = wrapStrikethrough(item, el);
```
Comment (iefserge) [inline lib/text.js]: I'd put it before the mark

File: lib/text.js (hunk 1 above)
Comment (iefserge) [inline lib/text.js]: wrapping ternary in parens should be good for readability (I think airbnb style requires it)

File: lib/text.js (hunk 2 above)
Comment (ellell) [inline lib/text.js]: oh right, thanks. perhaps even before the href?

**Pull request merged**